Library/Formula/stack.rb

```diff
@@ -0,0 +1,15 @@
+class Stack < Formula
+  desc "A cross-platform program for developing Haskell projects."
+  homepage "https://github.com/commercialhaskell/stack"
+  url "https://github.com/fpco/stack/releases/download/v0.0.0-beta/stack-0.0.0-x86_64-osx.gz"
```

Comment:
We've been shipping binaries for most packages for a couple of years now. Most users get binaries delivered, provided that a) they use the default install options, b) they haven't told Homebrew to build from source, and c) the binary is compatible with their OS version and Homebrew installation location.

Thanks!